angular-frontend/AUTHORIZATION.md

```diff
@@ -71,6 +73,8 @@ spec:
     - name: message.write
 ```
 
+__Note:__ You can find a matchLabel for your `AuthServer` by running `kubectl get authserver <the name of your authserver> -n <the namespace where your authserver is> -oyaml | yq .metadata.labels`.
```

Comment:
**[suggestion]**: "You can find a matchLabel" --> "You can find labels for your `AuthServer`"